app/src/main/res/values/strings.xml

```diff
@@ -24,7 +24,11 @@
   <string name="state_end_exploration_button">RETURN TO TOPIC</string>
   <string name="state_learn_again_button">LEARN AGAIN</string>
   <string name="topic_overview_see_more">See More</string>
-  <string name="topic_download_text">(%s MB)</string>
+  <string name="topic_download_text"> (%s MB)</string>
```

Comment:
Why this change? Please remove the extra space.

Reply:
Removed